tests/react/useReducer_hook.js

```diff
@@ -32,12 +32,13 @@ const initialState = { count: 0 };
 }
 
 {
-  const initialAction = {
-    type: "reset",
-    payload: 123
-  };
+  function init(arg){
+    return {
+      count: arg.payload
+    }
+  }  
 
-  const [state, dispatch] = React.useReducer(reducer, initialState, initialAction);
+  const [state, dispatch] = React.useReducer(reducer, { type: "reset", payload: 123 }, init);
```

Comment:
Not super important, but why does the init arg in this example have a type and payload? Implies that it's the same type as the action, but that's not necessarily true.

Reply:
No reason, just translated what was already there. I’ll change it. I specifically wanted it to *not* have the signature of the state, but I should have gone with something totally different. Thx!